ui/src/components/canvas.module.css

```diff
@@ -0,0 +1,27 @@
+.result-status__success{
```

Comment:
As I commented [here](https://github.com/codepod-io/codepod/pull/59#pullrequestreview-1187057820), let's inline the styles.